homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -298,7 +299,9 @@ class AutomowerSensorEntityDescription(SensorEntityDescription):
         key="next_start_timestamp",
         translation_key="next_start_timestamp",
         device_class=SensorDeviceClass.TIMESTAMP,
-        value_fn=lambda data: dt_util.as_local(data.planner.next_start_datetime),
+        value_fn=lambda data: convert_timestamp_to_datetime_utc(
+            data.planner.next_start_timestamp, dt_util.DEFAULT_TIME_ZONE
+        ),
```

Comment:
How can a timestamp be timezone aware?

Reply:
Can you use `dt_util.utc_from_timestamp` here?

Reply:
The API declares this as "local timestamp" of the device. So timestamp `0` is not January 1st 1970 in UTC. Instead it's January 1st 1970 in the timezone of the device.

Reply:
That horrible.

Would you please add a comment to that effect

Reply:
Yes it's really hard to handle. I've added a comment.